src/main/java/com/delivery/common/exception/ExceptionController.java

```diff
@@ -0,0 +1,38 @@
+package com.delivery.common.exception;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+/**
+ *  @ControllerAdvice : 모든 컨트롤러에서 발생하는 예외를 잡아 처리해주는 어노테이션
+ */
+@ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
를 이용하면 더 간결히 처리할 수 있을텐데 @ControllerAdvice를 이용한 이유가 있을까요?

Reply:
간결하다기보다는 RestControllerAdvice가 RestBody + ControllerAdvice를 포함하고 있는데요. 구현할 당시 RestControllerAdvice가 생각나지 않아, ControllerAdvice로 했습니다. ControllerAdvice로 바꾸면 될 것 같습니다.